pkg/operator/configobservation/network/observe_network.go

```diff
@@ -0,0 +1,104 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
+
+	"github.com/golang/glog"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Use `"github.com/ghodss/yaml"` instead (returns `map[string]interface{}` instead of `map[interface{}]interface{}`).